src/style/app.module.css

```diff
@@ -3027,38 +3096,72 @@ form > input {
 }
 
 .email_button {
+  --bs-btn-active-bg: var(--search-button-bg);
+  --bs-btn-active-border-color: var(--search-button-bg);
+  --bs-btn-hover-bg: var(--search-button-bg);
+  --bs-btn-hover-border-color: var(--search-button-bg);
   position: absolute;
   z-index: 10;
   bottom: 0;
   right: 0;
   height: 100%;
   display: flex;
   background-color: var(--search-button-bg);
-  border-color: var(--search-button-border, var(--black-color));
+  border-color: var(--search-button-bg);
   justify-content: center;
   align-items: center;
+  color: var(--brown-color);
+}
+.email_button:hover {
+  color: var(--brown-color) !important;
+  box-shadow: var(--hover-shadow);
 }
 
 .login_btn {
-  background-color: var(--search-button-bg);
-  border-color: var(--search-button-border, var(--black-color));
+  font-weight: bold;
+  color: var(--brown-color);
+  --bs-btn-bg: var(--search-button-bg);
+  --bs-btn-border-color: var(--search-button-bg);
+  --bs-btn-hover-bg: var(--search-button-bg);
+  --bs-btn-hover-border-color: var(--search-button-bg);
+  --bs-btn-active-color: var(--search-button-bg);
+  --bs-btn-active-bg: var(--search-button-bg);
+  --bs-btn-active-border-color: var(--search-button-bg);
+  --bs-btn-disabled-bg: var(--search-button-bg);
+  --bs-btn-disabled-border-color: var(--search-button-bg);
   margin-top: 1rem;
   margin-bottom: 1rem;
   width: 100%;
   transition: background-color 0.2s ease;
   cursor: pointer;
 }
 
+.login_btn:hover {
+  color: var(--brown-color) !important;
+  box-shadow: var(--hover-shadow);
+}
+
 .reg_btn {
-  background-color: var(--dropdown-border-color);
-  border-color: var(--dropdown-border-color);
+  font-weight: bold;
+  background-color: var(--grey-light);
+  border-color: var(--grey-light);
+
+  --bs-btn-hover-bg: var(--grey-light);
+  --bs-btn-hover-border-color: var(--grey-light);
+  --bs-btn-active-color: var(--grey-light);
+  --bs-btn-active-bg: var(--grey-light);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`